examples/python/notebook/blueprint.ipynb

```diff
@@ -1,128 +1,132 @@
 {
-    "cells": [
```

Comment:
looks like my code formatter has a different opinion here than the last person who touched it

Reply:
same for me, so I guess this is the correct one